lib/assembly-image/image.rb

```diff
@@ -118,12 +129,14 @@ def dpg_jp2_filename
     #   puts derivative_image.path # '/input/path_to_file.jp2'
     # rubocop:disable Metrics/AbcSize
     # rubocop:disable Metrics/MethodLength
+    # rubocop:disable Metrics/PerceivedComplexity:
+    # rubocop:disable Metrics/CyclomaticComplexity:
```

Comment:
I think you could avoid this complexity with a guard clause here. But I'd be happy to see anyone elses thoughts.

Reply:
Was able to reduce some complexity with @jcoyne's suggestion; however, after looking more closely at the code guard clause wouldn't help since not exiting the method.